test/async-hooks/init-hooks.js

```diff
@@ -212,13 +212,14 @@ class ActivityCollector {
 }
 
 exports = module.exports = function initHooks({
-    oninit,
-    onbefore,
-    onafter,
-    ondestroy,
-    allowNoInit,
-    logid,
-    logtype } = {}) {
+  oninit,
+  onbefore,
+  onafter,
+  ondestroy,
+  allowNoInit,
+  logid,
+  logtype
```

Comment:
I was trying to propose this in the docs, but it seems this lacks approvement:
https://github.com/nodejs/node/pull/12557